proj/package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.18.0",
+    "firebase": "^5.5.5",
```

Comment:
not supposed to be here for this PR. commit changes that are only related to this PR